src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -79,9 +79,9 @@ azure-mgmt-sql==0.26.0
 azure-mgmt-sqlvirtualmachine==0.5.0
 azure-mgmt-storage==17.0.0
 azure-mgmt-trafficmanager==0.51.0
-azure-mgmt-web==0.48.0
-azure-mgmt-synapse==0.6.0
-azure-multiapi-storage==0.6.0
+azure-mgmt-web==2.0.0
+azure-mgmt-synapse==0.3.0
```

Comment:
just curious, why downgrading this version? It might need live test of synapse tests also.

Reply:
This was a change from rebasing to the upstream/dev before starting PR